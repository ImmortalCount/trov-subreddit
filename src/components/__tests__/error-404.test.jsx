import React from 'react'
import renderer from 'react-test-renderer'
import Error404 from '../error-404'

describe('Error404 Component', () => {
    const defaultProps = {
        location: {
            pathname: 'foo'
        }
    }

    it('Error404 matches snapshot', () => {
        const modalRender = renderer
            .create(<Error404 {...defaultProps} />)
            .toJSON()
        expect(modalRender).toMatchSnapshot()
    })
})
