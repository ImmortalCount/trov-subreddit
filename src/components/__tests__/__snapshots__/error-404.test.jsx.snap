// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 Component Error404 matches snapshot 1`] = `
<div>
  There is no content at 
   foo
</div>
`;
